src/SplitIO/Sdk/Validator/flagSetsValidator.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace SplitIO\Sdk\Validator;
+
+use SplitIO\Component\Utils as SplitIOUtils;
+use SplitIO\Split as SplitApp;
+
+const REG_EXP_FLAG_SET = "/^[a-z0-9][_a-z0-9]{0,49}$/";
+
+class FlagSetsValidator
+{
+    public static function areValid($flagSets, $operation)
+    {
+        if (!is_array($flagSets) || SplitIOUtils\isAssociativeArray($flagSets) || count($flagSets) == 0) {
+            SplitApp::logger()->error($operation . ': FlagSets must be a non-empty list.');
+            return array();
+        }
+
+        $sanitized = [];
+        foreach ($flagSets as $flagSet) {
+            if ($flagSet == null) {
+                continue;
+            }
+
+            if (!is_string($flagSet)) {
+                SplitApp::logger()->error($operation . ': FlagSet must be a string and not null. ' .
+                $flagSet . ' was discarded.');
+                continue;
+            }
+            $sanitizedFlagSet = self::sanitize($flagSet, $operation);
+            if (!is_null($sanitizedFlagSet)) {
+                array_push($sanitized, $sanitizedFlagSet);
+            }
+        }
+
+        return array_values(array_unique($sanitized));
+    }
+
+    private static function sanitize($flagSet, $operation)
+    {
+        $trimmed = trim($flagSet);
```

Comment:
missing string check

Reply:
this can solve it https://github.com/splitio/php-client/blob/master/src/SplitIO/Sdk/Validator/InputValidator.php#L46

Reply:
`toString` is deprecated https://github.com/splitio/php-client/blob/master/src/SplitIO/Sdk/Validator/InputValidator.php#L44C6-L44C6